client-v2/src/components/FrontsEdit/ArticleFragmentFormInline.tsx

```diff
@@ -545,14 +550,17 @@ class FormComponent extends React.Component<Props, FormComponentState> {
   };
 
   private handleImageChange: EventWithDataHandler<React.ChangeEvent<any>> = (
-    e,
+    e: ValidationResponse | undefined,
     ...args: [any?, any?, string?]
```

Comment:
What's being passed in here is a `ValidationResponse` object with a src property etc, not an event. What the ConditionalField component expects the signature of the onChange function to be is `EventWithDataHandler<React.ChangeEvent<any>>` : 
```
Type '(e: ValidationResponse | undefined, args_0?: any, args_1?: any, args_2?: string | undefined) => void' is not assignable to type 'EventWithDataHandler<ChangeEvent<any>>'.
  Types of parameters 'e' and 'event' are incompatible.
    Type 'ChangeEvent<any> | undefined' is not assignable to type 'ValidationResponse | undefined'.
      Type 'ChangeEvent<any>' is missing the following properties from type 'ValidationResponse': src, thumb, origin, height, width
```

Reply:
For posterity -- after a chat IRL, this has been resolved by the `unknown` type.

As far as we can tell, redux-form's typings for its `Field` component aren't quite right -- they expect an event in an `onChange` handler, which doesn't match the [current documentation.](https://redux-form.com/7.4.0/docs/api/field.md/#-code-onchange-event-newvalue-previousvalue-name-gt-void-code-optional-) Until they're correct, we patch this by casting the incoming value.